CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## v0.63 [2023-04-20]
+
+_Breaking changes_
+
+- The `NIST Cybersecurity Framework (CSF) v1.1` benchmarks has been updated to better align with [AWS Audit Manager NIST Cybersecurity Framework (CSF) v1.1](https://docs.aws.amazon.com/audit-manager/latest/userguide/NIST-Cybersecurity-Framework-v1-1.html) framework. The following updates have been made: ([#604](https://github.com/turbot/steampipe-mod-aws-compliance/pull/604))
```

Comment:
@madhushreeray30 do we have a link to the official guide of NIST benchmark? If yes could you please add that?